src/account/interface.ts

```diff
@@ -219,4 +219,21 @@ export abstract class AccountInterface extends ProviderInterface {
     estimateFeeAction: EstimateFeeAction,
     details: EstimateFeeDetails
   ): Promise<BigNumberish>;
+
+  /**
+   * Get starknet.id stark name with the address
```

Comment:
nitpick: new line here

```suggestion
   * Get starknet.id stark name with the address
   *
```